src/utils/AppUtils.ts

```diff
@@ -0,0 +1,10 @@
+import {Request, Response, NextFunction} from "express"
+
+/**
+ * Simply renders a view template.
+ *
+ * @param template the template name
+ */
+export const renderTemplate = (template: string) => (req: Request, res: Response, next: NextFunction) => {
+    return res.render(template, {templateName: template});
```

Comment:
What is `templateName` used for? Is that needed? In other projects I used to call `res.render(templateFileName)` directly where `templateFileName` had simple string value e.g. `index`. `Whole call was really as simple as `res.render('index')`.